### DISCUSSION TITLE
Is it possible to get VAD results faster?

### DISCUSSION THREAD

**Discussion by Purfview** [pyannote/pyannote-audio#1695] [Q&A]
Is it possible to get VAD results faster, for example [pyannote-onnx ](https://github.com/pengzhendong/pyannote-onnx) implementation is ~5 times faster for me on CPU?

**Comment by hbredin** [comment]
I'd love to help you with that but could *you* prepare a self-contained Google Colab notebook showing the performance discrepancy between the two so that I can have a look?
